### PR TITLE
ci(tests): update ios version for tests

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **TYPE**

This PR resolves []

### Description

- updating the version of ios that the tests run. they are not broken, but now they are a bit more up-to-date than before.
- also, i added the tiny yarnrc line, to make sure that when we run `yarn add x`, the version will be installed like `"x": "1.2.3"` rather than `"x": "^1.2.3"`. we are trying to move to exact versions pinned.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -1 +1,2 @@
-unsafe-disable-integrity-migration true
\ No newline at end of file
+unsafe-disable-integrity-migration true
+save-prefix ""
```
Comment (MounirDhahri) [inline .yarnrc:2]: 👍

**Review by MounirDhahri — DISMISSED** [review]
Thanks 🙌

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 394c332ee584fd8226ed5e441adb5db88edf4b3f
</p>

**Comment by MounirDhahri** [comment]
#squahsongreen

**Comment by MounirDhahri** [comment]
wow! peril took two days to merge the PR 😄

**Comment by pvinis** [comment]
ah no no. it was red, I fixed it this morning.

**Pull request merged**